MACOS.md

```diff
@@ -198,14 +184,67 @@ brew upgrade openssl     || brew install openssl
 ```
 
 
-## Sublime Text 3 - Your text editor
+## Visual Studio Code
+
+### Installation
+
+Let's install [Visual Studio Code](https://code.visualstudio.com) text editor.
+
+Copy (`CMD` + `C`) the command below then paste it in your terminal (`CMD` + `V`):
+
+```bash
+brew install --cask visual-studio-code
+```
+
+Then launch VS Code:
+
+```bash
+'/Applications/Visual Studio Code.app'
+```
+
+Now, add VS Code to your Dock by right-clicking on the icon to bring up the context menu and choosing Options, Keep in Dock.
```

Comment:
Shall we add a screenshot 👇 ❔ 

<img width="566" alt="Screen Shot 2021-04-04 at 19 07 27" src="https://user-images.githubusercontent.com/8090140/113516161-0efd6100-9579-11eb-8f4f-48adfeaa6e82.png">

Reply:
:ok_hand: :heavy_check_mark: